modules/components/public/views/partials/atoms/button.liquid

```diff
@@ -97,5 +99,6 @@ metadata:
   {% if href and disabled != true %}href="{{href}}"{% endif %}
   {% if tag == 'button' and disabled %}disabled{% endif %}
   {% if tag == 'button' %}type="{{type}}"{% endif %}
+  {% if buttonDisabled != blank %}:disabled="{{buttonDisabled}}"{% endif %}
```

Comment:
Why? You can apply any js logic in your parent partial, or use the `attributes` param if you need custom attributes